packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -61,65 +49,162 @@ test('natMathHelpers', t => {
   );
   t.throws(
     () =>
-      coerce(
-        harden({ brand: { getAllegedName: () => 'somename' }, value: 4n }),
+      m.coerce(
+        {
+          brand: harden({
+            getAllegedName: () => 'somename',
+            isMyIssuer: async () => false,
+            getDisplayInfo: () => ({}),
+          }),
+          value: 4n,
+        },
+        mockBrand,
       ),
     {
-      message: /The brand in the allegedAmount .* in 'coerce' didn't match the amountMath brand/,
+      message: /The brand in the allegedAmount .* in 'coerce' didn't match the specified brand/,
     },
     `coerce can't take the wrong brand`,
   );
   t.throws(
-    () => coerce(3),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    () => m.coerce(3n, mockBrand),
+    {
+      message: `The brand in amount (a bigint) doesn't look like a brand. Did you pass a value rather than an amount?`,
+    },
     `coerce needs a brand`,
   );
+});
+
+test('natMathHelpers coerce no brand', t => {
+  t.throws(
+    // @ts-ignore
+    () => m.coerce(m.make(4n, mockBrand)),
+    {
+      message:
+        "The brand (an undefined) doesn't look like a brand. Did you pass a value rather than an amount?",
+    },
+    `brand is required in coerce`,
+  );
+});
+
+test('natMathHelpers getValue', t => {
+  t.is(m.getValue(m.make(4n, mockBrand), mockBrand), 4n);
```

Comment:
I'd write a parallel test for `t.is(m.getValue(m.make(4, mockBrand), mockBrand), 4n)`

Reply:
Done